docs/README.md

```diff
@@ -1,9 +1,67 @@
 # Documentation
+
 ## Overview
-This directory contains the following documents that relate to the project:
 
-- [Installation](https://github.com/kyma-project/helm-broker/blob/master/docs/installation.md) describes how to install the Helm Broker.
-- [Configuration](https://github.com/kyma-project/helm-broker/blob/master/docs/configuration.md) describes the environment variables you can configure.
-- [Development](https://github.com/kyma-project/helm-broker/blob/master/docs/development.md) describes how to develop the project.
-- [Releasing](https://github.com/kyma-project/helm-broker/blob/master/docs/releasing.md) describes the Helm Broker release process.
-- [Example usage](https://github.com/kyma-project/helm-broker/blob/master/docs/example-usage.md) describes how to provision a Redis instance using the Helm Broker and addons.
+This directory contains Helm Broker documentation.
```

Comment:
not really needed

Reply:
Actually, I'd keep this sentence and remove the whole next paragraph.